lib/l10n/app_es.arb

```diff
@@ -1,3 +1,182 @@
 {
-  "helloWorld": "Hola Mundo!"
-}
\ No newline at end of file
+  "welcome": "Bienvenido",
```

Comment:
This will require further review, we need to check the grammar and the length of the copies:
<img width="404" alt="Screenshot 2023-08-28 at 11 26 59" src="https://github.com/witnet/my-wit-wallet/assets/16032491/f7db90fb-8d58-4de0-bc72-9e7bcae140f7">

Reply:
yeah 😅 
i updated it so "es" is disabled / commented out in the constants-
the app will try to use the locale set on the device, if it is not supported-> it will default to "en"

Reply:
We aren't going to support multiple languages in the next release. So we could use this file as a starting point for translating the wallet to Spanish.